### PR TITLE
Removing trailing whitespaces

### PR DESCRIPTION
**Pull request by kvprasoon**
Removing trailing whitespaces

### CODE REVIEW

File: src/Microsoft.PowerShell.SHiPS/Resources/Resource.resx [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (jianyunt) [inline src/Microsoft.PowerShell.SHiPS/Resources/Resource.resx]: These are auto-generated code. Can you please revert it?

File: src/Microsoft.PowerShell.SHiPS/Resources/Resource.Designer.cs [hunk 2]
```diff
@@ -10,8 +10,8 @@
 
 namespace Microsoft.PowerShell.SHiPS.Resources {
```
Comment (jianyunt) [inline src/Microsoft.PowerShell.SHiPS/Resources/Resource.Designer.cs]: This is auto-generated code. Can you please revert it?

**Review by jianyunt — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.SHiPS/Resources/Resource.resx (hunk 1 above)
Comment (kvprasoon) [reply src/Microsoft.PowerShell.SHiPS/Resources/Resource.resx]: didn't get you, is it about reverting the whole change in Resource.resx file

File: src/Microsoft.PowerShell.SHiPS/Resources/Resource.Designer.cs (hunk 2 above)
Comment (kvprasoon) [reply src/Microsoft.PowerShell.SHiPS/Resources/Resource.Designer.cs]: didn't get you, is it about reverting the whole change in Resource.Designer.cs file

File: src/Microsoft.PowerShell.SHiPS/Resources/Resource.resx (hunk 1 above)
Comment (jianyunt) [reply src/Microsoft.PowerShell.SHiPS/Resources/Resource.resx]: Yes please revert all changes made in this .resx  file.

File: src/Microsoft.PowerShell.SHiPS/Resources/Resource.Designer.cs (hunk 2 above)
Comment (jianyunt) [reply src/Microsoft.PowerShell.SHiPS/Resources/Resource.Designer.cs]: Yes please revert all changes made in this Resource.Designer.cs file.

**Pull request merged**